scripts/device/xcvu13p-fhgb2104-2-i.tcl

```diff
@@ -0,0 +1,36 @@
+set ::cfg [dict create]
```

Comment:
add support for VU9P

Reply:
More information is needed about VU9P, including specific chip models and schematics.